__docs__/wonder-blocks-cell/detail-cell-testing-snapshots.stories.tsx

```diff
@@ -179,6 +179,11 @@ export const Scenarios: Story = {
                     rightAccessory: undefined,
                 },
             },
+            {
+                name: "Right to Left",
+                props: {...defaultProps, title: longTextWithNoWordBreak},
+                decorator: <div dir="rtl" />,
+            },
```

Comment:
<img width="661" height="94" alt="Image" src="https://github.com/user-attachments/assets/fdce1539-41de-4746-a0e6-79bd473d1185" />

thought (for later): I wonder if there's something we can do so that chevron left/right arrow icons will automatically respond to the language direction.

One idea is to componentize the arrow icon so that it'll automatically flip based on the `dir`, so consumers don't have to repeat this logic everywhere! We could also componentize any other icons that we expect to change with language direction. They could live in `wonder-blocks-icon`, similar to the `GemIcon` and `StreakIcon` components.